### ISSUE TITLE
for 13K images 3.5GB .pkl file generated

### ISSUE THREAD

**Issue by vijaysagi1** [davidsandberg/facenet#687]
I had downloaded lfw dataset from the web i trained with the below commands

The following the commands i used marjorly

step 1)
python  train_softmax.py  --models_base_dir /home//Desktop/facenet/ --data_dir /home/Desktop/facenet/src/align/output --image_size 160 --model_def models.inception_resnet_v1  --optimizer RMSPROP --learning_rate -1 --max_nrof_epochs 5 --keep_probability 0.8 --random_crop --random_flip --learning_rate_schedule_file /home/Desktop/facenet/data/learning_rate_schedule_classifier_casia.txt --weight_decay 5e-5 --center_loss_factor 1e-2 --center_loss_alfa 0.9

Step 2)
python classifier.py     --mode TRAIN  --data_dir /root/facenet/src/align --model /root/facenet/20180404-202256 --classifier_filename /root/class.pkl --test_data_dir /root/facenet/src/align/

FYI
learning_rate_schedule_classifier_casia.txt 
# Maps an epoch number to a learning rate
0:  0.1
65: 0.01
77: 0.001
1000: 0.000

The .pkl generated is really huge pls suggest what best can be done to reduce it so that i can inference within 1-2 secs with best possible accuracy?

Thanks
vijay

**Comment by maxisme** [comment]
Can you quantify 'really huge'?

**Comment by Yaoxingtian** [comment]
the  same problem as  me ,   did you solve it   ?
